.github/workflows/ci.yml

```
@@ -10,7 +10,7 @@ jobs:
     name: Ruby ${{ matrix.ruby }}
     strategy:
       matrix:
-        ruby: ["2.6", "2.7", "3.0", "3.1", "3.2"]
+        ruby: ["2.6", "2.7", "3.0", "3.1", "3.2", "3.3"]
     steps:
       - uses: actions/checkout@v3
```

While you're at it (prevents node deprecation warnings)

```suggestion
      - uses: actions/checkout@v4
```

---

Thanks, changed. Can you point me where can I see this deprecation?

---

Yeah, sure. Here's the run from when you opened this PR: https://github.com/SamSaffron/memory_profiler/actions/runs/9553006533

You can either go there from the Repo Actions tab and look for it there, or click on "Checks" for this PR and then "CI" to go to the summary tab.

GitHub doesn't make these notices very obvious, the amount of worklflows that still use v2/v3 is astronomical.

---

Ah, thanks. I looked in the wrong place originally (each job details) and haven't noticed any.